### PR TITLE
feat(centerpoint): release CenterPoint 2.2.0

### PR DESCRIPTION
**Pull request by KSeangTan**
## Summary
Release the following CenterPoint version:
- `base/v2.2.0`:
    - Add a new training set: `db_jpntaxi_gen2_v1`
- `J6Gen2/v2.2.1`
   - Finetune from `CenterPoint base/2.2.0`
   - Include intensity as an extra feature
- `JPNTaxi_Gen2/v2.2.1`
   - Finetune from `CenterPoint base/2.2.0` with `db_jpntaxi_gen2` datasets only
   - Include intensity as an extra feature

## Perfomance 
- For more details, please take a look at [TIER IV Internal link](https://docs.google.com/spreadsheets/d/13Stt9hdbTER6ugaRMEZscbt_6kD7ld-0b30JyeTGTrs/edit?usp=sharing)

#### base
| eval range: 120m         | mAP   | car (152,572) | truck (24,172) | bus (9,839) | bicycle (5,317) | pedestrian (50,699) |
|--------------------------|-------|----------------|----------------|-------------|------------------|----------------------|
| CenterPoint base/2.2     | **66.00** | **82.60**     | **55.50**     | **71.90**   | **56.40**       | 66.50               |
| CenterPoint base/2.1     | 65.55 | 82.51           | 53.12          | 68.95       | 56.32           | **66.81**           |

#### JPNTaxi_Gen2
| eval range: 120m                | mAP   | car (8,571) | truck (3,349) | bus (4,148) | bicycle (310) | pedestrian (8,665) |
|--------------------------------|-------|--------------|----------------|--------------|----------------|---------------------|
| CenterPoint base/2.2           | **57.01** | **86.30**   | 35.50         | **60.80**    | 30.50         | **72.10**           |
| CenterPoint base/2.1           | 53.44 | 85.00         | 32.50         | 47.00        | **33.40**      | 69.30               |
| CenterPoint JPNTaxi_Gen2/2.2.1 | 50.90 | 84.20         | **35.60**     | 53.70        | 12.00         | 68.90               |

#### J6gen2
| eval range: 120m              | mAP   | car (44,008) | truck (2,471) | bus (1,464) | bicycle (333) | pedestrian (6,459) |
|------------------------------|-------|---------------|----------------|--------------|----------------|---------------------|
| CenterPoint base/2.1         | 72.40 | 83.60         | 54.40          | 84.00        | 74.30          | 65.70               |
| CenterPoint base/2.1         | 71.83 | 83.51         | 53.01          | 84.62        | 73.42          | 64.58               |
| CenterPoint base/2.0         | 71.04 | 83.56         | 53.08          | 85.06        | 68.54          | 64.97               |
| CenterPoint base/1.7         | 70.10 | 82.27         | 49.65          | 81.78        | 73.67          | 63.14               |
| CenterPoint J6Gen2/2.2.1     | **75.20** | 84.70     | **57.40**      | **85.50**    | 80.70          | **67.80**           |
| CenterPoint J6Gen2/2.1.1     | 74.24 | 84.65         | 54.66          | 85.45        | 79.07          | 67.35               |
| CenterPoint J6Gen2/2.0.1     | 74.60 | **84.83**     | 56.48          | 83.70        | **81.06**      | 66.91               |
| CenterPoint J6Gen2/1.7.1     | 72.35 | 83.34         | 53.26          | 82.13        | 79.10          | 63.92               |

#### LargeBus
| eval range: 120m              | mAP   | car (13,831) | truck (2,137) | bus (95)  | bicycle (724) | pedestrian (3,916) |
|------------------------------|-------|---------------|----------------|-----------|----------------|---------------------|
| CenterPoint base/2.2         | 71.68 | 88.60         | 63.60          | 78.70     | 64.10          | 63.40               |
| CenterPoint base/2.1         | 71.85 | 88.85         | 64.67          | 80.35     | 62.15          | 63.20               |
| CenterPoint base/2.0         | 71.09 | 89.01         | 64.11          | 77.75     | 61.04          | 63.56               |
| CenterPoint base/1.7         | 69.15 | 87.87         | 53.81          | 78.40     | 63.08          | 62.58               |
| CenterPoint J6Gen2/2.2.1     | 74.67 | 90.20         | **70.60**      | 78.20     | 66.50          | 67.80               |
| CenterPoint J6Gen2/2.1.1     | **74.99** | 90.35     | 70.43          | 79.63     | 67.51          | 66.99               |
| CenterPoint J6Gen2/2.0.1     | 74.73 | **90.53**     | 69.55          | **80.06** | 65.72          | **67.82**           |
| CenterPoint J6Gen2/1.7.1     | 72.17 | 89.36         | 60.87          | 78.40     | **67.71**      | 64.84               |

#### JPNTaxi
| eval range: 120m         | mAP   | car (16,142) | truck (4,578) | bus (1,457) | bicycle (1,040) | pedestrian (11,971) |
|--------------------------|-------|----------------|----------------|--------------|-------------------|------------------------|
| CenterPoint base/2.2     | **66.43** | 76.00     | 52.70         | **74.60**    | 61.00            | **68.00**              |
| CenterPoint base/2.1     | 65.46 | 75.86         | **52.90**     | 71.79        | 59.39            | 67.36               |
| CenterPoint base/2.0     | 66.20 | **76.02**     | 52.59         | 71.18        | **63.55**        | 67.67               |
| CenterPoint base/1.7     | 65.86 | 75.46         | 51.65         | 73.10        | 61.25            | 67.82               |

#### J6
| eval range: 120m         | mAP   | car (67,551) | truck (10,013) | bus (2,503) | bicycle (2,846) | pedestrian (19,117) |
|--------------------------|-------|----------------|------------------|---------------|-------------------|------------------------|
| CenterPoint base/2.2     | 65.08 | **82.20**     | 54.10            | 72.40         | 53.10            | 63.60               |
| CenterPoint base/2.1     | 67.19 | 82.15         | 55.54            | 77.13         | 54.42            | **66.66**              |
| CenterPoint base/2.0     | **67.41** | 82.09     | **56.08**        | 80.41         | 53.75            | 64.71               |
| CenterPoint base/1.7     | 67.35 | 81.28         | 52.10            | **83.28**     | **56.22**         | 63.90               |

## Overview 
- `J6Gen2/2.2.1`: Highest mAP in J6gen2 (75.20)
- Performance almost similar for `2.2.x` and `2.1.x` models
- `JPNTaxi_Gen2/2.2.1` is worse than `base/2.2.0`, likely the training and test set are not diverse, and the model is overffited to the training set

### CODE REVIEW

File: projects/CenterPoint/docs/CenterPoint/v2/base.md [hunk 1]
```diff
@@ -41,6 +61,7 @@
 
 | eval range: 120m         | mAP  | car <br> (44,008) | truck <br> (2,471) | bus <br> (1,464) | bicycle <br> (333) | pedestrian <br> (6,459) |
 | -------------------------| ---- | ----------------- | ------------------- | ---------------- | -------------------- | ------------------------ |
+| CenterPoint base/2.1     | 72.40 | 83.60 | 54.40 | 84.00 | 74.30   | 65.70       |
```
Comment (SamratThapa120) [inline projects/CenterPoint/docs/CenterPoint/v2/base.md:64]: [TYPO] `base/2.2`

File: projects/CenterPoint/docs/CenterPoint/v2/jpntaxi_gen2.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Deployed model for CenterPoint J6Gen2/1.X
```
Comment (SamratThapa120) [inline projects/CenterPoint/docs/CenterPoint/v2/jpntaxi_gen2.md:1]: [TYPO] `JPNTAXI-GEN2/2.X`

**Review by SamratThapa120 — APPROVED** [review]
LGTM. 
Thanks, left some minor comments.

File: projects/CenterPoint/docs/CenterPoint/v2/base.md (hunk 1 above)
Comment (KSeangTan) [reply projects/CenterPoint/docs/CenterPoint/v2/base.md:64]: Nice catch, thanks!
